docs/topics/feed-exports.rst

```diff
@@ -233,6 +234,20 @@ The serialization format to be used for the feed. See
 
 .. setting:: FEED_EXPORT_FIELDS
 
+FEED_EXPORT_ENCODING
+--------------------
+
+Default: ``None``
+
+The encoding to be used for the feed.
+
+If unset or set to ``None`` (default) it uses UTF-8 for everything except JSON output,
+which uses safe numeric encoding (``\uXXXX`` sequences) for historic reasons.
+
+Use ``utf-8`` if you want UTF-8 for JSON too.
+
+.. setting:: FEED_EXPORT_ENCODING
```

Comment:
Sorry, I had missed this:
it should be `.. setting:: FEED_EXPORT_FIELDS` here and `.. setting:: FEED_EXPORT_ENCODING` earlier